src/scripts/profiling/run_profiling.py

```diff
@@ -268,13 +279,31 @@ def run_profiling(
                 f"\tWas        : {additional_stats['html_output']}"
                 f"\tReplaced by: {output_html_file}"
             )
-        additional_stats["html_output"] = str(output_html_file)
+        additional_stats["html_output"] = str(output_html_file.name)
 
     if write_pyisession:
         output_ipysession_file = output_dir / f"{output_name}.pyisession"
         print(f"Writing {output_ipysession_file}", end="...", flush=True)
         scale_run_session.save(output_ipysession_file)
         print("done")
+        
+    if write_flat_html:
+        if not ANSI2HTML_AVAILABLE:
+            raise ValueError("ansi2html required for flat HTML output.")
```

Comment:
```suggestion
            print("ansi2html required for flat HTML output - SKIPPING.")
```

For robust-ness (and not to waste the ~8 hours or so a profiling session takes) I've tended to not throw errors when something is unavailable, so I'd prefer to just print a message to STDOUT here and then skip the rest of the block.

That way we don't interrupt the workflow, and the usual profiling output will be pushed to the profiling repo in the workflow step that follows. If we throw an error here, the profiling workflow will stop before uploading the results that have already been rendered to the profiling site.

Reply:
That's a very good point - probably it would make sense to either just print a message as you say or fail early by checking `ansi2html` is available at the beginning of the function and raising an error there. I have a slight preference for latter as I think failing loudly is generally preferably to a message which might get missed. Will update accordingly!